components/admin_console/manage_tokens_modal/manage_tokens_modal.tsx

```diff
@@ -14,58 +16,41 @@ import Avatar from 'components/widgets/users/avatar';
 
 import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
 
-export default class ManageTokensModal extends React.PureComponent {
-    static propTypes = {
+type Props = {
```

Comment:
Please add the removed comments for the props

Reply:
thanks for the hint, forgot to add them back! fixed now ;-)